prow/gerrit/adapter/adapter.go

```diff
@@ -182,7 +182,11 @@ func (c *Controller) ProcessChange(instance string, change client.ChangeInfo) er
 		return fmt.Errorf("failed to get SHA from base branch: %v", err)
 	}
 
-	triggeredJobs := []string{}
+	type triggeredJob struct {
+		name   string
+		report bool
+	}
+	triggeredJobs := []triggeredJob{}
```

Comment:
You are not initializing anything so use `var triggeredJobs []triggeredJob`

Reply:
Thank you for the review. Fixed.